src/tactic/omega/clause.lean

```diff
@@ -0,0 +1,56 @@
+/-
+Copyright (c) 2019 Seul Baek. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Author: Seul Baek
+
+A tactic for discharging Presburger arithmetic goals using the Omega test.
+-/
+
+import tactic.omega.term
+
+@[reducible] def clause := (list term) × (list term)
+
+namespace clause
+
+def holds (v) : clause → Prop 
+| (eqs,les) :=
+  ( (∀ t : term, t ∈ eqs → 0 = term.val v t) 
+    ∧ (∀ t : term, t ∈ les → 0 ≤ term.val v t) )
+
+def sat (c : clause) : Prop :=
+  ∃ v : nat → int, holds v c
+
+def unsat (c : clause) : Prop := ¬ c.sat 
+
+def append (c1 c2 : clause) : clause :=
+(c1.fst ++ c2.fst, c1.snd ++ c2.snd)
+
+def holds_append {v c1 c2} : 
```

Comment:
add the type of `v`, `c1` and `c2`. Also use `c₁` and `c₂` instead of `c1` and `c2`